internal/bminventory/inventory_test.go

```diff
@@ -7296,7 +7278,7 @@ var _ = Describe("DownloadMinimalInitrd", func() {
 			InfraEnv: models.InfraEnv{
 				ID:            &clusterID,
 				PullSecretSet: true,
-				Type:          common.ImageTypePtr(models.ImageTypeMinimalIso),
+				Type:          common.ImageTypePtr(models.ImageTypeFullIso),
```

Comment:
I think it would be good to have a sanity test case for both types.
I think it would be fine to put the validation code here into a helper function then have an example for each iso type.